### PR TITLE
Tt user story 18

### PR DESCRIPTION
**Pull request by tylertomlinson**
## All Submissions:

* [x] Merge the target branch into the PR branch. Fix any conflicts that might appear.
* [x] Have you checked to ensure that other pull request are approved for merge [Pull Requests](../../../pulls)?
* [x] What stories does the PR include

### New Feature Submissions:

* [x] Have you added an explanation of what your changes do?
* [x] Have you lint your code locally prior to submission?

### Testing:

* [x] Have you written new tests for your core changes, as applicable?
TESTING THIS STORY WAS GNARLY
* [x] Have you successfully ran tests with your changes locally?
* [x] All new and existing tests passed.

**Test coverage** = 100%
--------------------------------------------------------------------------------
### Change Log
Created methods on Item model
`most_popular`
`least_popular`
`quantity_bought`

Testing method on model level as well as view level(within items index page)

### CODE REVIEW

File: app/views/items/index.html.erb [hunk 1]
```diff
@@ -1,25 +1,45 @@
...
 <% else %>
-  <h1>All Items</h1>
+<h1>All Items</h1>
 <% end %>
 <section class="grid-container">
+  <aside class="grid-item">
+    <div id='most-popular-items'>
+      <h2>Most Popular Items</h2>
+      <ul>
+        <% @items.most_popular(5).each do |item| %>
```
Comment (danielpselph) [inline app/views/items/index.html.erb:12]: Well done on getting this put in and making links

File: app/views/items/index.html.erb [hunk 2]
```diff
@@ -1,25 +1,45 @@
...
+        <h4><%=link_to item.name %> </h4>
+        <li>Quantity Ordered: <%= item.quantity_ordered %></li>
```
Comment (alex-latham) [inline app/views/items/index.html.erb:14]: I like the links

**Review by danielpselph — APPROVED** [review]
changes approved

**Review by alex-latham — APPROVED** [review]
Looks good

**Pull request merged**